src/__tests__/ceramic_integration.test.ts

```diff
@@ -162,6 +162,7 @@ async function makeCAS(
   configCopy.ceramic.apiUrl = 'http://localhost:' + minConfig.ceramicPort
   configCopy.blockchain.connectors.ethereum.network = 'ganache'
   configCopy.blockchain.connectors.ethereum.rpc.port = minConfig.ganachePort + ''
+  configCopy.useSmartContractAnchors = true
```

Comment:
I think we still want it false by default right?  Ideally we'd actually run all these tests twice, once with it on and once with it off.  But if we're only running them once for now we should run the version that is most similar to what we run in prod

Reply:
Yes I agree. Smart contract anchors are required in js-ceramic for ganache test networks with chainid 1337. This requirement is turned on in js-ceramic through a block-threshold that isn't exposed. While I can mock the function so we can skip this verification, doing so turns off all other verifications that we probably want in our tests. So i'll need to make another PR to js-ceramic to add a new chainId to specifically tests non-smart contract anchors.

Reply:
yeah that sounds like a problem in js-ceramic.  The smart contract anchoring project is still in progress, it should be disabled by default and we should have a flag to turn it on in our tests until we're ready to turn it on by default.